src/Avalonia.Controls/ComboBox.cs

```diff
@@ -290,26 +290,12 @@ protected override void OnPointerWheelChanged(PointerWheelEventArgs e)
         protected override void OnPointerPressed(PointerPressedEventArgs e)
         {
             base.OnPointerPressed(e);
-            if(!e.Handled && e.Source is Visual source)
+            if(e.Handled)
             {
-                if (_popup?.IsInsidePopup(source) == true)
-                {
-                    e.Handled = true;
-                    return;
-                }
-            }
-
-            if (IsDropDownOpen)
```

Comment:
Removing this will break `OverlayDismissEventPassThrough`, as mentioned in the original code comment.